Dockerfile

```diff
@@ -24,3 +25,6 @@ RUN ln -s /usr/bin/openshift-builder /usr/bin/openshift-sti-build && \
 LABEL io.k8s.display-name="OpenShift Builder" \
       io.k8s.description="This is a component of OpenShift and is responsible for executing image builds." \
       io.openshift.tags="openshift,builder"
+
+RUN mkdir /var/cache/blobs
+VOLUME /var/lib/containers/cache
```

Comment:
Is the intent that `/var/lib/containers/cache` will be mounted in by the build controller (from the host)?

Reply:
it is, but i don't think we gain anything by declaring this volume, i'd rather leave it out.  We mount stuff in lots of places in the image that aren't explicitly declared as volumes.

@nalind was there a specific reason you needed to add the volume declaration?

Reply:
It's a hint that there's a benefit to mounting `/var/lib/containers/cache` as a shared volume, but you're right that we don't do that for the other things that we mount in, so I'll drop the line.